go/api/glide_cluster_client.go

```diff
@@ -69,3 +75,102 @@ func (client *GlideClusterClient) CustomCommand(args []string) (ClusterValue[int
 	}
 	return CreateClusterValue(data), nil
 }
+
+// CustomCommandWithRoute executes a single command, specified by args, without checking inputs. Every part of the command,
+// including the command name and subcommands, should be added as a separate value in args. The returning value depends on
+// the executed command.
+//
+// See [Valkey GLIDE Wiki] for details on the restrictions and limitations of the custom command API.
+//
+// Parameters:
+//
+//	args  - Arguments for the custom command including the command name.
+//	route - Specifies the routing configuration for the command. The client will route the
+//	        command to the nodes defined by route.
+//
+// Return value:
+//
+//	The returning value depends on the executed command and route.
+//
+// For example:
+//
+//	route := config.SimpleNodeRoute(config.RandomRoute)
+//	result, err := client.CustomCommand([]string{"ping"}, route)
+//	result.Value().(string): "PONG"
```

Comment:
API of `ClusterValue` was changed recently
```suggestion
//	result.SingleValue().(string): "PONG"
```

Reply:
done